source/locale/pt_BR/LC_MESSAGES/faq.po

```diff
@@ -746,10 +759,10 @@ msgstr "Como eu posso adicionar um novo tipo de nó no menu?"
 #: ../../source/faq/faq.rst:419
 msgid ""
 "There is a tutorial on how to add new nodes and two examples of node too, check "
-"the `flowbroker library`_ for more details."
+"the `GitHub flowbroker repository`_ for more details."
 msgstr ""
 "Há um tutorial sobre como adicionar novos nós e dois exemplos de nós também, "
-"verifique a `flowbroker library`_  para mais detalhes."
+"verifique o repositório GitHub flowbroker para mais detalhes."
```

Comment:
There was a link markup here before, it should be there, right?

markup: \`link\`_
example:
> [...] \`flowbroker library\`_ [...]

Reply:
I kept it as requested in the review. However, I will post a link insertion note for the next fix